patterns/Architectual/Model_View_ViewModel/style.css

```diff
@@ -0,0 +1,39 @@
+body{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100svh;
+}
+
+#mvvm {
+  text-align: center;
+  margin-top: 50px;
+}
+#mvvm h1 {
+  padding-bottom: 64px;
+}
+
+#mvvm button {
+  padding: 15px 30px; /* 按钮大一些 */
```

Comment:
Can you translate this comment and the others to English? Or remove if they're not necessary.

Reply:
sure , they was used for todo notes to me 😂 so they are not necessary